### PR TITLE
feat(external): adding batch_key and labels_key

### PR DESCRIPTION
**Pull request by mys721tx**
Adds batch_key and labels_key per documentation. Also corrected the reference to scvi.external.

### CODE REVIEW

**Review by canergen — CHANGES_REQUESTED** [review]
Thanks for the contribution. LGTM. Please add a release note and a test for both fields.

File: tests/external/scar/test_scar.py [hunk 1]
```diff
@@ -7,8 +7,11 @@
 def test_scar():
     n_latent = 5
     adata = synthetic_iid()
+
+    adata.obs["batch"] = adata.obs["batch"].cat.rename_categories(["batch_2", "batch_3"])
+
     adata.X = scipy.sparse.csr_matrix(adata.X)
-    SCAR.setup_anndata(adata)
+    SCAR.setup_anndata(adata, batch_key="batch")
```
Comment (canergen) [inline tests/external/scar/test_scar.py:14]: Please add label_key="batch". It's good to verify that handling label_key is not broken in the future.

Comment (mys721tx) [reply tests/external/scar/test_scar.py]: Looks like some other test cases the renaming is not strictly necessary. Should we keep that?

**Review by mys721tx — COMMENTED** [review]
Both `batch_key` and `labels_key` are added to the test case.

**Pull request merged**